src/Management/src/EndpointCore/Mappings/MappingsEndpointMiddleware.cs

```diff
@@ -266,6 +270,9 @@ private ApiDescriptionProviderContext GetApiDescriptions(IReadOnlyList<ActionDes
                 return new ApiDescriptionProviderContext(new List<ActionDescriptor>());
             }
 
+#if NETCOREAPP3_0
+            throw new NotImplementedException("ApiDescriptionActionData is `internal` in AspNetCore 3");
```

Comment:
@rynowak this is another spot that breaks for us in 3.0. I'm sure we'll get to something that works, but if you have a suggested alternative it'd be nice to save some research time

Reply:
What's the purpose of this method? What is this trying to do?

Reply:
If you're just trying to access the API Descriptions - use `IApiDescriptionCollectionProvider`.  This looks really scary to me though, it looks like you're duplicating functionality.

Reply:
This method exists to return the MVC and WebAPI Routes and Route templates used by the application, specifically for compatibility with a feature in Apps Manager on PCF

![image](https://user-images.githubusercontent.com/3947063/64633034-60a72f00-d3c8-11e9-8dd3-b456c4ec8bf8.png)

Reply:
You should be able to get this data from `IApiDescriptionCollectionProvider`.

Reply:
Good to know, I'll have to try that out to see if there's a reason we have it in the way it is now instead...